orange/src/main/java/com/hartwig/hmftools/orange/report/chapters/GermlineFindingsChapter.java

```diff
@@ -107,6 +109,13 @@ private void addGermlineDeletions(@NotNull Document document) {
             String title = "Potentially pathogenic germline deletions (" + reportableGermlineGainsLosses.size() + ")";
```

Comment:
Maybe add "homozygous" to title to make it more distinct compared to heterozygous?

Reply:
I changed the title of the germline LOH section to be more distinct instead.